examples/native-fungible/src/contract.rs

```diff
@@ -0,0 +1,323 @@
+// Copyright (c) Zefchain Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+#![cfg_attr(target_arch = "wasm32", no_main)]
+
+mod state;
+
+use self::state::NativeFungibleToken;
+use async_trait::async_trait;
+use fungible::{ApplicationCall, FungibleResponse, Message, Operation};
+use linera_sdk::{
+    base::{Account, AccountOwner, Amount, Owner, SessionId, WithContractAbi},
+    contract::system_api,
+    ApplicationCallOutcome, CalleeContext, Contract, ExecutionOutcome, MessageContext,
+    OperationContext, SessionCallOutcome, ViewStateStorage,
+};
+use native_fungible::TICKER_SYMBOL;
+use thiserror::Error;
+
+linera_sdk::contract!(NativeFungibleToken);
+
+impl WithContractAbi for NativeFungibleToken {
+    type Abi = fungible::FungibleTokenAbi;
+}
+
+#[async_trait]
+impl Contract for NativeFungibleToken {
+    type Error = Error;
+    type Storage = ViewStateStorage<Self>;
+
+    async fn initialize(
+        &mut self,
+        _context: &OperationContext,
+        state: Self::InitializationArgument,
+    ) -> Result<ExecutionOutcome<Self::Message>, Self::Error> {
+        // Validate that the application parameters were configured correctly.
+        assert!(Self::parameters().is_ok());
```

Comment:
This line is redundant now; the next one seems to also check whether it's `Ok`.)